composer.json

```diff
@@ -37,7 +37,8 @@
     ],
     "require": {
         "php": ">= 8",
-        "loophp/iterators": "^2.3.1"
+        "loophp/iterators": "^2.3.1",
+        "vimeo/psalm": "dev-master#3560ef771743048302599aea151e206a50f4a744"
```

Comment:
Psalm 5.4 is out now, is this still needed? 

https://github.com/vimeo/psalm/releases/tag/5.4.0

Reply:
We need the next version, 5.4.1 ;)